splunk_add_on_ucc_framework/global_config.py

```diff
@@ -123,6 +123,24 @@ def settings(self) -> List[Any]:
                 settings.append(tab)
         return settings
 
+    @property
+    def logging_tab(self) -> Dict[str, Any]:
+        for tab in self.tabs:
+            if tab.get("type") == "loggingTab":
+                return tab
+            elif all(
```

Comment:
I am not sure if storing conditions for logging tabs in two places would be ok.

As `self.tabs` after expanding is a list of Tab (+LoggingTab) objects, you could just do:

```python
@property
def logging_tab(self) -> Dict[str, Any]:
    for tab in self.tabs:
        if isinstance(tab, LoggingTab):
            return tab
    return {}
```

Alternatively, you could call `from_definition` method once again:

```python
@property
def logging_tab(self) -> Dict[str, Any]:
    for tab in self.tabs:
        if LoggingTab.from_definition(tab) is not None:
            return tab
    return {}
```

Reply:
The `from_definition` method worked for old and new format. I've updated the code.